site/gatsby-site/src/pages/cite/edit.js

```diff
@@ -92,13 +101,36 @@ function EditCitePage(props) {
 
   const incident_ids = incidentsData?.incidents.map((incident) => incident.incident_id);
 
-  const loading = loadingIncident || loadingReport;
+  const loading = loadingIncident || loadingReport || reportTranslations === null;
 
   const [linkReportsToIncidents] = useMutation(LINK_REPORTS_TO_INCIDENTS);
 
   const addToast = useToastContext();
 
-  const { config } = useLocalization();
+  useEffect(() => {
+    if (reportData?.report) {
+      const translations = availableLanguages
+        .map((c) => c.code)
+        .reduce((acc, languageCode) => {
+          // Find existing translation for this language
+          const existingTranslation = reportData.report.translations?.find(
+            (t) => t.language === languageCode
+          );
+
+          // If translation exists use its values, otherwise use empty values
+          acc[`translations_${languageCode}`] = existingTranslation
```

Comment:
@kepae This `translations_[LANGUAGE CODE]` format is actually to be used in the `IncidentReportForm.js` form input field names. It is not used in the data base structure, is just for the form data.